lib/apm.js

```diff
@@ -461,6 +461,8 @@ var Instrumentation = function(core, options, callback) {
               reply: [{ok:1}]
             };
 
+            func.apply(this, arguments);
```

Comment:
This seems to be the issue - the command object is created, but the original function is not invoked.